tests/test_workspace.py

```diff
@@ -108,3 +110,10 @@ def test_output(self):
         self.assertIn('messages', keys)
```

Comment:
could it be possible to simply add another assert In line here for stderr instead of a separate test function?